collections/_posts/2021-09-10-important-version-1.2.6-information.en.md

```diff
@@ -0,0 +1,25 @@
+
+Unfortunately, this caused nodes all over to receive a large quantity of compressed blocks, and the work of swapping the increased volume of compressed blocks into the DB for the uncompressed versions caused some nodes to be unable to keep up, and experience higher-than-acceptable memory usage. As a result, we brought those Blueboxes offline and developed this update. To address this problem, this update contains better handling logic on the node side to manage that incoming data in a way that allows the node to maintain stability, which should prevent this from happening again.
+
+As the chain is continuing to grow and the bloat still needs to be addressed, Chia still intends to ramp back up to this same number of temporary Bluebox Timelords to compress things down for you and streamline the db size, but this time we will do a gradual increase of them over several weeks starting next week. As a result, anyone who has not updated to [version 1.2.6](https://github.com/Chia-Network/chia-blockchain/releases/tag/1.2.6) will potentially experience issues with their node if it’s queuing is unable to handle the rate of updates… while we would rather you did not experience this pain at all, we feel there is a greater user pain in the size of the current DB and the time it takes to sync fresh, and we are motivated to deploy this temporary cluster to bring it down for you.
+
+## So what _is_ a Bluebox Timelord?
+
+As you may be aware, a [Timelord](https://github.com/Chia-Network/chia-blockchain/wiki/Timelords) is a server that generates the next block in the chain, with the fastest running one at any given moment being the authoritative server moving the chain forward. The Timelord infuses a winning proof of space into the blockchain, and then the process repeats. In order to do its job and do it well, a Timelord must be _fast_, at the expense of many other things including efficiency. This means the blocks it generates are uncompressed and ugly in the interest of speed. Enter the Bluebox… there can be any number of Blueboxes running, and their job is to scour the blockchain db and look for old blocks that are still uncompressed. They then clean up and compress a block, and then gossip it around to all of those nodes, who take this compressed block and perform a few confirmations, and then swap out the old bloated blocks in their DB for the new streamlined one. 
```

Comment:
Timelords don't generate the next block, they generate proofs of time, which are a required part of the block.

Reply:
A block is a data structure which incldues a bunch of things:
- Proof of space
- Proof of time (VDF)
- Signature
- Transactions

The timelord only generates one piece

Reply:
Proof of time are large and take up a lot of space. Blueboxes look for large proofs of time and compress them.

Reply:
"perform a few confirmations", should be changed to: "verify the proof of time".
Confirmations means something else in blockchain (this transaction has 5 confirmations means that there were 5 blocks mined after that transactino was included).

Reply:
updated!